### PR TITLE
Migrate user docs to Docs repo

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

The source repo should only contain docs for contributing to or maintaining the project. User-focused documentation has been moved to the MicrosoftDocs/PowerShell-Docs repo.

## PR Context

Removed files from the `docs/learning-powershell` folder and update the FAQ.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/7.3/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and opened an issue in the relevant tool repository. This may include:
        - [ ] Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode
        (which runs in a different PS Host).
            - [ ] Issue filed: 
        - [ ] Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
            - [ ] Issue filed: 
        - [ ] Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
            - [ ] Issue filed: 
        - [ ] Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).
            - [ ] Issue filed:

### CODE REVIEW

**Comment by sdwheeler** [comment]
@SteveL-MSFT We talked about moving this content in Shiproom.

**Review by daxian-dbw — APPROVED** [review]
LGTM

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: docs/FAQ.md [hunk 1]
```diff
@@ -2,89 +2,86 @@
...
 
 ## Why didn't an error throw an exception?
 
-Error handling in PowerShell is a bit weird, as not all errors result in catchable exceptions by default.
-Setting `$ErrorActionPreference = 'Stop'` will likely do what you want;
-that is, cause non-terminating errors instead to terminate.
-Read [An Introduction To Error Handling in PowerShell][error] for more information.
-
-[error]: https://gist.github.com/TravisEz13/9bb811c63b88501f3beec803040a9996
+Error handling in PowerShell is a bit weird, as not all errors result in catchable exceptions by
```
[the remark this answers is not in the document]
Comment (sdwheeler) [reply docs/FAQ.md:34]: Fixed

**Review by andyleejordan — APPROVED** [review]
LGTM! And I double-checked the links.

**Review by SteveL-MSFT — APPROVED** [review]

**Pull request merged**